Casks/h/hdfview.rb

```diff
@@ -5,11 +5,11 @@
   url "https://support.hdfgroup.org/ftp/HDF5/releases/HDF-JAVA/hdfview-#{version}/bin/HDFView-#{version}-macos11_64.tar.gz"
   name "HDFView"
   desc "Tool for browsing and editing HDF files"
-  homepage "https://confluence.hdfgroup.org/display/support"
+  homepage "https://www.hdfgroup.org/downloads/hdfview/"
 
   livecheck do
-    url "https://confluence.hdfgroup.org/display/support/Download+HDFView"
-    regex(/HDFView\+(\d+(?:\.\d+)+)/i)
+    url :homepage
+    regex(/href=.*?HDFView[._-]v?(\d+(?:\.\d+)+)[._-]macos11_64\.(?:t|zip)/i)
```

Comment:
```suggestion
    regex(/href=.*?HDFView[._-]v?(\d+(?:\.\d+)+)[._-]macos[^"' >]*?\.(?:t|zip)/i)
```

This will fail to match if/when upstream updates the `macos` suffix (e.g., `macos12_64` instead of `macos11_64`), so we may be better off glossing over the specifics after `macos`.